Languages/IronPython/IronPython/Runtime/PythonDictionary.cs

```diff
@@ -28,7 +28,7 @@
 
 namespace IronPython.Runtime {
 
-    [PythonType("dict"), Serializable, DebuggerTypeProxy(typeof(PythonDictionary.DebugProxy)), DebuggerDisplay("dict, {Count} items")]
+    [PythonType("dict"), Serializable, DebuggerTypeProxy(typeof(PythonDictionary.DebugProxy)), DebuggerDisplay("Count = {Count}")]
```

Comment:
For two reasons:
* To make it consistent with  other uses of `[DebuggerDisplay]` within the .NET Framework
* The specification covers the _Value_ column in the variables view. The type is already displayed in the _Type_ column, no need to repeat it.